server.js

```diff
@@ -7,7 +7,9 @@ const mysql = require('mysql');
 const app = express();
 
 const selectAllUsers = 'SELECT * FROM users';
-const selectAllEvents = `SELECT id, team_name, DATE_FORMAT(date, "%M %d") as date, 
+const selectTeamNames = 'SELECT name FROM team';
```

Comment:
Consider moving the SQL statements to their respective functions instead of clustering them on top of the file.